README.rst

```diff
@@ -80,14 +81,17 @@ The equivalent code in Lua:
        },
    }
 
+Note: The line with `---@type rainbow_delimiters.config` is optional, but it
+will give you better completions with LuaLS.
+
```

Comment:
I don't think we need to mention this. It is obvious that comments are always optional, and anyone using this plugin is expected to know at least that much Lua.

Reply:
Okay, I have removed it now. I am often unsure what exactly one expects the users to know.